lib/coverage-instrumenter.js

```diff
@@ -22,6 +24,9 @@ CoverageInstrumenter.prototype.processString = function(content, relativePath) {
     noAutoWrap: true
   });
 
+  // fixes the path so the lcov.info can be read by 3rd parties (coveralls)
+  relativePath = pathFixer.fixPath(relativePath, this._app);
```

Comment:
in my use-case I fixed the paths directly with my provider: https://github.com/codecov/support/blob/master/site/docs.md#-fixing-paths